File: RepoTransBench/source_projects/Python/Matoking_protontricks/public_tests/test_public_util.py
from protontricks.util import lower_dict

def test_public_lower_dict():
    # Use different keys and values,
    # all keys are strings for robust comparison since util.lower_dict expects keys to have .lower()
    # see: https://github.com/Matoking/protontricks/blob/master/src/protontricks/util.py#L54
    upper_dict = {"KEY": 10, "ALPHA": 20, "Z": "VALUE", "MiXeD": "Flag", "foo": "Bar"}
    lowered = lower_dict(upper_dict)
    assert lowered == {
        "key": 10,
        "alpha": 20,
        "z": "VALUE",
        "mixed": "Flag",
        "foo": "Bar",
    }

File: RepoTransBench/target_projects/Python/Rust/Matoking_protontricks/tests/original/test_public_util.rs
use protontricks::util::lower_dict_flat;
use std::collections::HashMap;

#[test]
fn test_public_lower_dict() {
    let mut upper_dict = HashMap::new();
    upper_dict.insert("KEY", 10);
    upper_dict.insert("ALPHA", 20);
    upper_dict.insert("Z", "VALUE");
    upper_dict.insert("MiXeD", "Flag");
    upper_dict.insert("foo", "Bar");
    let lowered = lower_dict_flat(&upper_dict);
    assert_eq!(lowered.get("key"), Some(&10));
    assert_eq!(lowered.get("alpha"), Some(&20));
    assert_eq!(lowered.get("z"), Some(&"VALUE"));
    assert_eq!(lowered.get("mixed"), Some(&"Flag"));
    assert_eq!(lowered.get("foo"), Some(&"Bar"));
}